nuget/bootstrap.nuspec

```diff
@@ -3,22 +3,19 @@
   <metadata>
     <id>bootstrap</id>
     <!-- pulled from package.json -->
-    <version>4</version>
+    <version>5</version>
     <title>Bootstrap CSS</title>
     <authors>The Bootstrap Authors, Twitter Inc.</authors>
     <owners>bootstrap</owners>
     <description>The most popular front-end framework for developing responsive, mobile first projects on the web.</description>
     <releaseNotes>https://blog.getbootstrap.com/</releaseNotes>
-    <summary>Bootstrap framework in CSS. Includes JavaScript</summary>
+    <summary>Bootstrap framework in CSS. Includes JavaScript but requires PopperJS which must be installed independently</summary>
```

Comment:
Can't the users use our bundle file? That way they shouldn't need to install popper manually.

Reply:
True, I forgot about the bundle. Docs cover this pretty well so just reverting the note.